optionlab/engine.py

```diff
@@ -91,30 +92,26 @@ def __init__(self, inputs: Inputs):
             else:
                 n_discarded_days = 0
 
-            self.__days2target = (
+            self.days_to_target = (
                 inputs.target_date - inputs.start_date
             ).days - n_discarded_days
         else:
-            self.__days2target = inputs.days_to_target_date
+            self.days_to_target = inputs.days_to_target_date
 
         for i, strategy in enumerate(inputs.strategy):
-            self.__type.append(strategy.type)
+            self.type.append(strategy.type)
 
             if isinstance(strategy, OptionStrategy):
-                self.__strike.append(strategy.strike)
-                self.__premium.append(strategy.premium)
-                self.__n.append(strategy.n)
-                self.__action.append(strategy.action)
-                self.__prevpos.append(strategy.prev_pos or 0.0)
+                self.strike.append(strategy.strike)
```

Comment:
There is _ missing in self.strike. It should be self._strike.

Reply:
It is used in plot.py, therefore not private anymore

Reply:
Well, here's an example of why I truly appreciate private attributes + @property-decorated methods, and why I've been using them extensively into all the new Python code I've been writing. I like to have strict control over how and even whether users of the lib access the internal variables that determine an object's state. By exposing direct user access to those variables, they might manage to achieve unexpected behavior from the code that I haven't anticipated myself. If doing this isn't considered a good practice in modern Python's OOP programming, I sincererly wonder why...

Reply:
What I would propose here is to not have private or protected attributes then, as there's no logical reason to expose "strike" outside of the class and not expose "n".